activestorage/lib/active_storage/attached/model.rb

```diff
@@ -40,7 +40,14 @@ module Attached::Model
       #     has_one_attached :avatar, service: :s3
       #   end
       #
-      def has_one_attached(name, dependent: :purge_later, service: nil)
+      # If you need to enable +strict_loading+ to raise an error when N+1 query happens,
```

Comment:
It does more than this, we usually say in the docs that it raise for lazily loaded queries.